rbi/yast/uishortcuts.rbi

```diff
@@ -0,0 +1,152 @@
+# typed: strong
+module Yast
+  module UIShortcuts
```

Comment:
this looks quite fragile, maybe generate it by script? Also we basically kill type checking here as each type has own limitation for params

Reply:
The way forward is to add more type information here, so generating this is a dead end.

So why did I include what appears to be a useless list?
It's because UIShortcuts is a mixin, and when I remove this RBI file, any `typed: true` class that includes it will fail on unknown methods. (seen in yast2.rpm)